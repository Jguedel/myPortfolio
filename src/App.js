import "./App.css";
import { BrowserRouter, Route, Routes } from "react-router-dom";
import Error404 from "./Pages/Error_404";
import HomePage from "./Pages/Home";
import ProjectList from "./Pages/ProjectList";
import ContactMe from "./Pages/ContactMe";

function App() {
  return (
    <div className="App">
      <BrowserRouter>
        <Routes>
          <Route exact path="/myPortfolio" element={<HomePage />} />
          <Route exact path="/Projects" element={<ProjectList />} />
          <Route exact path="/ContactMe" element={<ContactMe />} />
          <Route exact path="/*" element={<Error404 />} />
        </Routes>
      </BrowserRouter>
    </div>
  );
}

export default App;
