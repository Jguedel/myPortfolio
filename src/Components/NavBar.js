import Container from "react-bootstrap/Container";
import Nav from "react-bootstrap/Nav";
import Navbar from "react-bootstrap/Navbar";
import { LinkContainer } from "react-router-bootstrap";
import "bootstrap/dist/css/bootstrap.min.css";
import "./Style/NavBar.css";

const NavBar = () => {
  return (
    <div className="sticky-top NavBar">
      <Navbar expand="lg" bg="dark" data-bs-theme="dark">
        <Container>
          <LinkContainer to="/myPortfolio">
            <Navbar.Brand href="/">Jacobs Porfolio</Navbar.Brand>
          </LinkContainer>
          <Navbar.Toggle aria-controls="basic-navbar-nav" />
          <Navbar.Collapse id="basic-navbar-nav">
            <Nav className="me-auto">
              <LinkContainer to="/Projects">
                <Nav.Link>Projects</Nav.Link>
              </LinkContainer>
              <LinkContainer to="/ContactMe">
                <Nav.Link>Contact Me</Nav.Link>
              </LinkContainer>
            </Nav>
          </Navbar.Collapse>
        </Container>
      </Navbar>
    </div>
  );
};

export default NavBar;
