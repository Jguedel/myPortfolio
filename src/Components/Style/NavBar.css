.bold{
    font-weight: bolder;
}
.NavBar a{
    padding-left: 5px;
    padding-right: 5px;
}
.NavBar a:hover{
    color:whitesmoke;
}
.active {
    font-weight:700;
  }

.navbar-brand{
    font-size: 30px;
}