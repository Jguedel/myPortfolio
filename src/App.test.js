import { render, screen } from "@testing-library/react";
import App from "./App";
import NavBar from "./Components/NavBar";

test("renders learn react link", () => {
  render(<App />);
  // expect(<NavBar />).toBeInTheDocument();
});
