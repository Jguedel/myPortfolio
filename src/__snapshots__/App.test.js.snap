// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Page Renders 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div>
          <div
            class="sticky-top NavBar"
          >
            <nav
              class="navbar navbar-expand-lg navbar-light bg-dark"
              data-bs-theme="dark"
            >
              <div
                class="container"
              >
                <a
                  class="navbar-brand"
                  href="/myPortfolio"
                >
                  Jacobs Porfolio
                </a>
                <button
                  aria-controls="basic-navbar-nav"
                  aria-label="Toggle navigation"
                  class="navbar-toggler collapsed"
                  type="button"
                >
                  <span
                    class="navbar-toggler-icon"
                  />
                </button>
                <div
                  class="navbar-collapse collapse"
                  id="basic-navbar-nav"
                >
                  <div
                    class="me-auto navbar-nav"
                  >
                    <a
                      class="nav-link"
                      data-rr-ui-event-key="/Projects"
                      href="/Projects"
                    >
                      Projects
                    </a>
                    <a
                      class="nav-link"
                      data-rr-ui-event-key="/ContactMe"
                      href="/ContactMe"
                    >
                      Contact Me
                    </a>
                  </div>
                </div>
              </div>
            </nav>
          </div>
          <div
            class="errorPage"
          >
            <div
              class="errorText"
            >
              <h1>
                <i
                  class="error404 fa-solid fa-triangle-exclamation"
                />
                 404 Error 
                <i
                  class="error404 fa-solid fa-triangle-exclamation"
                />
              </h1>
              <h4>
                This page does not exist
              </h4>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div>
      <div
        class="App"
      >
        <div>
          <div
            class="sticky-top NavBar"
          >
            <nav
              class="navbar navbar-expand-lg navbar-light bg-dark"
              data-bs-theme="dark"
            >
              <div
                class="container"
              >
                <a
                  class="navbar-brand"
                  href="/myPortfolio"
                >
                  Jacobs Porfolio
                </a>
                <button
                  aria-controls="basic-navbar-nav"
                  aria-label="Toggle navigation"
                  class="navbar-toggler collapsed"
                  type="button"
                >
                  <span
                    class="navbar-toggler-icon"
                  />
                </button>
                <div
                  class="navbar-collapse collapse"
                  id="basic-navbar-nav"
                >
                  <div
                    class="me-auto navbar-nav"
                  >
                    <a
                      class="nav-link"
                      data-rr-ui-event-key="/Projects"
                      href="/Projects"
                    >
                      Projects
                    </a>
                    <a
                      class="nav-link"
                      data-rr-ui-event-key="/ContactMe"
                      href="/ContactMe"
                    >
                      Contact Me
                    </a>
                  </div>
                </div>
              </div>
            </nav>
          </div>
          <div
            class="errorPage"
          >
            <div
              class="errorText"
            >
              <h1>
                <i
                  class="error404 fa-solid fa-triangle-exclamation"
                />
                 404 Error 
                <i
                  class="error404 fa-solid fa-triangle-exclamation"
                />
              </h1>
              <h4>
                This page does not exist
              </h4>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div>
        <div
          class="sticky-top NavBar"
        >
          <nav
            class="navbar navbar-expand-lg navbar-light bg-dark"
            data-bs-theme="dark"
          >
            <div
              class="container"
            >
              <a
                class="navbar-brand"
                href="/myPortfolio"
              >
                Jacobs Porfolio
              </a>
              <button
                aria-controls="basic-navbar-nav"
                aria-label="Toggle navigation"
                class="navbar-toggler collapsed"
                type="button"
              >
                <span
                  class="navbar-toggler-icon"
                />
              </button>
              <div
                class="navbar-collapse collapse"
                id="basic-navbar-nav"
              >
                <div
                  class="me-auto navbar-nav"
                >
                  <a
                    class="nav-link"
                    data-rr-ui-event-key="/Projects"
                    href="/Projects"
                  >
                    Projects
                  </a>
                  <a
                    class="nav-link"
                    data-rr-ui-event-key="/ContactMe"
                    href="/ContactMe"
                  >
                    Contact Me
                  </a>
                </div>
              </div>
            </div>
          </nav>
        </div>
        <div
          class="errorPage"
        >
          <div
            class="errorText"
          >
            <h1>
              <i
                class="error404 fa-solid fa-triangle-exclamation"
              />
               404 Error 
              <i
                class="error404 fa-solid fa-triangle-exclamation"
              />
            </h1>
            <h4>
              This page does not exist
            </h4>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Page Render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div>
          <div
            class="sticky-top NavBar"
          >
            <nav
              class="navbar navbar-expand-lg navbar-light bg-dark"
              data-bs-theme="dark"
            >
              <div
                class="container"
              >
                <a
                  class="navbar-brand"
                  href="/myPortfolio"
                >
                  Jacobs Porfolio
                </a>
                <button
                  aria-controls="basic-navbar-nav"
                  aria-label="Toggle navigation"
                  class="navbar-toggler collapsed"
                  type="button"
                >
                  <span
                    class="navbar-toggler-icon"
                  />
                </button>
                <div
                  class="navbar-collapse collapse"
                  id="basic-navbar-nav"
                >
                  <div
                    class="me-auto navbar-nav"
                  >
                    <a
                      class="nav-link"
                      data-rr-ui-event-key="/Projects"
                      href="/Projects"
                    >
                      Projects
                    </a>
                    <a
                      class="nav-link"
                      data-rr-ui-event-key="/ContactMe"
                      href="/ContactMe"
                    >
                      Contact Me
                    </a>
                  </div>
                </div>
              </div>
            </nav>
          </div>
          <div
            class="errorPage"
          >
            <div
              class="errorText"
            >
              <h1>
                <i
                  class="error404 fa-solid fa-triangle-exclamation"
                />
                 404 Error 
                <i
                  class="error404 fa-solid fa-triangle-exclamation"
                />
              </h1>
              <h4>
                This page does not exist
              </h4>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div>
        <div
          class="sticky-top NavBar"
        >
          <nav
            class="navbar navbar-expand-lg navbar-light bg-dark"
            data-bs-theme="dark"
          >
            <div
              class="container"
            >
              <a
                class="navbar-brand"
                href="/myPortfolio"
              >
                Jacobs Porfolio
              </a>
              <button
                aria-controls="basic-navbar-nav"
                aria-label="Toggle navigation"
                class="navbar-toggler collapsed"
                type="button"
              >
                <span
                  class="navbar-toggler-icon"
                />
              </button>
              <div
                class="navbar-collapse collapse"
                id="basic-navbar-nav"
              >
                <div
                  class="me-auto navbar-nav"
                >
                  <a
                    class="nav-link"
                    data-rr-ui-event-key="/Projects"
                    href="/Projects"
                  >
                    Projects
                  </a>
                  <a
                    class="nav-link"
                    data-rr-ui-event-key="/ContactMe"
                    href="/ContactMe"
                  >
                    Contact Me
                  </a>
                </div>
              </div>
            </div>
          </nav>
        </div>
        <div
          class="errorPage"
        >
          <div
            class="errorText"
          >
            <h1>
              <i
                class="error404 fa-solid fa-triangle-exclamation"
              />
               404 Error 
              <i
                class="error404 fa-solid fa-triangle-exclamation"
              />
            </h1>
            <h4>
              This page does not exist
            </h4>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Page Renders 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div>
          <div
            class="sticky-top NavBar"
          >
            <nav
              class="navbar navbar-expand-lg navbar-light bg-dark"
              data-bs-theme="dark"
            >
              <div
                class="container"
              >
                <a
                  class="navbar-brand"
                  href="/myPortfolio"
                >
                  Jacobs Porfolio
                </a>
                <button
                  aria-controls="basic-navbar-nav"
                  aria-label="Toggle navigation"
                  class="navbar-toggler collapsed"
                  type="button"
                >
                  <span
                    class="navbar-toggler-icon"
                  />
                </button>
                <div
                  class="navbar-collapse collapse"
                  id="basic-navbar-nav"
                >
                  <div
                    class="me-auto navbar-nav"
                  >
                    <a
                      class="nav-link"
                      data-rr-ui-event-key="/Projects"
                      href="/Projects"
                    >
                      Projects
                    </a>
                    <a
                      class="nav-link"
                      data-rr-ui-event-key="/ContactMe"
                      href="/ContactMe"
                    >
                      Contact Me
                    </a>
                  </div>
                </div>
              </div>
            </nav>
          </div>
          <div
            class="errorPage"
          >
            <div
              class="errorText"
            >
              <h1>
                <i
                  class="error404 fa-solid fa-triangle-exclamation"
                />
                 404 Error 
                <i
                  class="error404 fa-solid fa-triangle-exclamation"
                />
              </h1>
              <h4>
                This page does not exist
              </h4>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div>
        <div
          class="sticky-top NavBar"
        >
          <nav
            class="navbar navbar-expand-lg navbar-light bg-dark"
            data-bs-theme="dark"
          >
            <div
              class="container"
            >
              <a
                class="navbar-brand"
                href="/myPortfolio"
              >
                Jacobs Porfolio
              </a>
              <button
                aria-controls="basic-navbar-nav"
                aria-label="Toggle navigation"
                class="navbar-toggler collapsed"
                type="button"
              >
                <span
                  class="navbar-toggler-icon"
                />
              </button>
              <div
                class="navbar-collapse collapse"
                id="basic-navbar-nav"
              >
                <div
                  class="me-auto navbar-nav"
                >
                  <a
                    class="nav-link"
                    data-rr-ui-event-key="/Projects"
                    href="/Projects"
                  >
                    Projects
                  </a>
                  <a
                    class="nav-link"
                    data-rr-ui-event-key="/ContactMe"
                    href="/ContactMe"
                  >
                    Contact Me
                  </a>
                </div>
              </div>
            </div>
          </nav>
        </div>
        <div
          class="errorPage"
        >
          <div
            class="errorText"
          >
            <h1>
              <i
                class="error404 fa-solid fa-triangle-exclamation"
              />
               404 Error 
              <i
                class="error404 fa-solid fa-triangle-exclamation"
              />
            </h1>
            <h4>
              This page does not exist
            </h4>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`renders learn react link 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div>
          <div
            class="sticky-top NavBar"
          >
            <nav
              class="navbar navbar-expand-lg navbar-light bg-dark"
              data-bs-theme="dark"
            >
              <div
                class="container"
              >
                <a
                  class="navbar-brand"
                  href="/myPortfolio"
                >
                  Jacobs Porfolio
                </a>
                <button
                  aria-controls="basic-navbar-nav"
                  aria-label="Toggle navigation"
                  class="navbar-toggler collapsed"
                  type="button"
                >
                  <span
                    class="navbar-toggler-icon"
                  />
                </button>
                <div
                  class="navbar-collapse collapse"
                  id="basic-navbar-nav"
                >
                  <div
                    class="me-auto navbar-nav"
                  >
                    <a
                      class="nav-link"
                      data-rr-ui-event-key="/Projects"
                      href="/Projects"
                    >
                      Projects
                    </a>
                    <a
                      class="nav-link"
                      data-rr-ui-event-key="/ContactMe"
                      href="/ContactMe"
                    >
                      Contact Me
                    </a>
                  </div>
                </div>
              </div>
            </nav>
          </div>
          <div
            class="errorPage"
          >
            <div
              class="errorText"
            >
              <h1>
                <i
                  class="error404 fa-solid fa-triangle-exclamation"
                />
                 404 Error 
                <i
                  class="error404 fa-solid fa-triangle-exclamation"
                />
              </h1>
              <h4>
                This page does not exist
              </h4>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div>
        <div
          class="sticky-top NavBar"
        >
          <nav
            class="navbar navbar-expand-lg navbar-light bg-dark"
            data-bs-theme="dark"
          >
            <div
              class="container"
            >
              <a
                class="navbar-brand"
                href="/myPortfolio"
              >
                Jacobs Porfolio
              </a>
              <button
                aria-controls="basic-navbar-nav"
                aria-label="Toggle navigation"
                class="navbar-toggler collapsed"
                type="button"
              >
                <span
                  class="navbar-toggler-icon"
                />
              </button>
              <div
                class="navbar-collapse collapse"
                id="basic-navbar-nav"
              >
                <div
                  class="me-auto navbar-nav"
                >
                  <a
                    class="nav-link"
                    data-rr-ui-event-key="/Projects"
                    href="/Projects"
                  >
                    Projects
                  </a>
                  <a
                    class="nav-link"
                    data-rr-ui-event-key="/ContactMe"
                    href="/ContactMe"
                  >
                    Contact Me
                  </a>
                </div>
              </div>
            </div>
          </nav>
        </div>
        <div
          class="errorPage"
        >
          <div
            class="errorText"
          >
            <h1>
              <i
                class="error404 fa-solid fa-triangle-exclamation"
              />
               404 Error 
              <i
                class="error404 fa-solid fa-triangle-exclamation"
              />
            </h1>
            <h4>
              This page does not exist
            </h4>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
